app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
```

Comment:
There is an unnecessary empty line here. Consider removing it to keep the code clean and organized.